tool/npm_template/demo.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+const path = require("path");
+const validator = require('./index.js');
+
+const filename = __dirname + '/Box.gltf';
```

Comment:
If you do decide to make this a command-line utility that ships with the package, you would change this to be a command-line argument, and rename this file.  Then people could `npm install -g gltf-validator` followed by `gltf-validator myFile.glb` right from the shell.